packages/client/hmi-client/src/views/ModelView.vue

```diff
@@ -7,19 +7,21 @@ import useResourcesStore from '@/stores/resources';
 import { Project } from '@/types/Project';
 import { RouteName } from '@/router/routes';
 import { useTabStore } from '@/stores/tabs';
+import * as _ from 'lodash';
```

Comment:
```suggestion
import { isEmpty } from 'lodash';
```

this would be better and most people at Uncharted are familiar with _LoDash_.